winlogbeat/_meta/fields.common.yml

```diff
@@ -61,22 +62,27 @@
         The keywords are used to classify an event.
 
     - name: log_name
-      type: keyword
+      # This does not exist yet
```

Comment:
Do you have same example values what it contains?

Reply:
I don't. I'll try to check out the docs, or better, generate some events